sdks/python/apache_beam/transforms/display_test.py

```diff
@@ -22,8 +22,16 @@
 from datetime import datetime
 import unittest
 
+import hamcrest as hc
+
 import apache_beam as beam
-from apache_beam.transforms.display import HasDisplayData, DisplayData, DisplayDataItem
+from apache_beam.transforms.display import *
+
+
+def make_nspace_display_data(component):
```

Comment:
And on that note, I'll point out how having `DisplayDataMatchers` have been valuable in the Java SDK to lower the bar to writing concise display data tests. Some things that are difficult without them:

* Cannot easily assert on single part of display data. For example, if you have many tests of a PTransform that vary a single property to validate display data, each assertion will redundantly include the full set of display data items + properties. For example, see [`TextIOTest`](https://github.com/apache/incubator-beam/blob/master/sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOTest.java#L538) `testWriteDisplayData()`, `testWriteDisplayDataValidateThenHeader()`, `testWriteDisplayDataValidateThenFooter()`.
* Tests for sub-component display data break encapsulation. For example, in order to test that the `Read` transform properly includes display data for its `Source`, your assert must list all display data for the `Source`. Later updating the `Source` display data will also break the `Read` transform test. In Java we have a `includesDisplayDataFrom(..)` matcher. See [`ReadTest.testDisplayData()`](https://github.com/apache/incubator-beam/blob/master/sdks/java/core/src/test/java/org/apache/beam/sdk/io/ReadTest.java#L99)
* Equality-tests on the display data map means adding additional elements to the display data schema (like `path`) will break all tests. It's convenient to assert on only the fields that a PTransform actually sets and exclude others filled in by the SDK (namespace and path).